doc-tool/src/dotty/tools/dottydoc/core/DocASTPhase.scala

```diff
@@ -27,20 +24,20 @@ class DocASTPhase extends Phase {
   def phaseName = "docASTPhase"
 
   /** Build documentation hierarchy from existing tree */
-  def collect(tree: Tree)(implicit ctx: Context): Entity = {
+  def collect(tree: Tree)(implicit ctx: Context): List[Entity] = {
```

Comment:
Why did you change `Entity` to `List[Entity]`?

Reply:
As far as I can see there are only lists of size 0 or 1.

Reply:
That's to support multiple `@usecase`. See the commit messages of https://github.com/lampepfl/dotty/pull/4789/commits/c528d6ae0a552df29ce2aa131114375d0d60079b and https://github.com/lampepfl/dotty/pull/4789/commits/f1da1e155f58818615e412ea9330148ee3f3b0bc

This is where longer lists may be created: https://github.com/lampepfl/dotty/pull/4789/files#diff-5a9874880800484ed4b082c8d776c661R30